pkg/storaged/fsys-panel.jsx

```diff
@@ -142,8 +142,12 @@ export class FilesystemsPanel extends React.Component {
                         { title: "Stratis" },
                         { title: mount },
                         {
-                            title: <StorageUsageBar stats={[Number(fs.Used[0] && Number(fs.Used[1])), pool_total]}
-                                                    critical={1} total={total} offset={offsets[i]} />,
+                            title: (pool.Overprovisioning
+                                ? <StorageUsageBar stats={[Number(fs.Used[0] && Number(fs.Used[1])), pool_total]}
+                                                       critical={1} total={total} offset={offsets[i]} />
+                                : <StorageUsageBar stats={[Number(fs.Used[0] && Number(fs.Used[1])), Number(fs.Size)]}
+                                                       critical={0.95} />
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18867-20230816-095630-d3ce589c-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)